### PR TITLE
Added pcdm:hasRelatedObject

### PR DESCRIPTION
**Pull request by jcoyne**
See https://github.com/projecthydra-labs/hydra-pcdm/issues/14

### CODE REVIEW

File: models.rdf [hunk 1]
```diff
@@ -89,5 +89,14 @@
       <rdfs:subPropertyOf rdf:resource="http://www.w3.org/ns/ldp#contains"/>
       <rdfs:isDefinedBy rdf:resource="http://pcdm.org/models#"/>
     </rdf:Property>
-   
+    
+    <rdf:Property rdf:about="http://pcdm.org/models#hasRelatedObject">
+      <rdfs:label xml:lang="en">has related object</rdfs:label>
+      <rdfs:comment xml:lang="en">Links to an Object which is related to this Object but doesn't directly describe or represent it.</rdfs:comment>
+      <rdfs:domain rdf:resource="http://pcdm.org/models#Object"/>
+      <rdfs:domain rdf:resource="http://pcdm.org/models#Collection"/>
```
Comment (jcoyne) [inline models.rdf:97]: Is this domain correct (Collection & Object)?

Comment (acoburn) [inline models.rdf:97]: As far as I understand this, the domain should be `http://www.openarchives.org/ore/terms/Aggregation`

See, for example, `http://pcdm.org/models#hasMember`

But isn't this what `ore:aggregates` does? Why define a new property?

**Comment by jcoyne** [comment]
I'm not sure this is valid. I'm closing it.

**Comment by escowles** [comment]
Closing in favor of #4

**Pull request closed without merging**